src/components/CompanyPlacement/CompanyPlacement.tsx

```diff
@@ -42,12 +43,12 @@ function CompanyPlacement() {
   const placementsWithImage = placementsEdges.map(edge => {
     return {
       title: edge.node.title,
+      order: edge.node.order,
       images: edge.node.images?.map(image =>
         image ? getImage(imageMap[image]) : null,
       ),
     }
-  })
-  console.log(placementsWithImage)
+  }).sort((a, b) => (a.order ? a.order : 0) - (b.order ? b.order : 0));
```

Comment:
```suggestion
  }).sort((a, b) => (a.order ?? 0) - (b.order ?? 0));
```

let's clean this up a little bit